### DISCUSSION TITLE
getServerSideProps is not getting invoked in Production mode

### DISCUSSION THREAD

**Discussion by dinakar17** [vercel/next.js#41736] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

"next": "^12.3.1"

### What browser are you using? (if relevant)

Chrome 106.0.5249.119

### How are you deploying your application? (if relevant)

Vercel

### Describe the Bug

I have the following code in **pages/index** file 
`export const getServerSideProps: GetServerSideProps = async () => {
  try {
    const res = await api.getRandomPosts();
    return {
      props: {
        data: res.data,
      },
    };
  } catch (err) {
    return {
      props: {
        error: "Unable to display posts at the moment. Please try again later",
      },
    };
  }
};`

So everything works really fine in development mode. But as soon as I've pushed my website to production through vercel getServerSideProps is not running as if it is not present.

I've searched the internet and have found people suggesting shifting from **getServerSideProps to useSWR** because of some payload issue. What's the point in using getServerSideProps if I did as they said?

### Expected Behavior

Expected Behaviour is 'getServerSideProps' should work in production too even for heavy payloads 

### Link to reproduction

Since the bug is in production I don't have any links

### To Reproduce

Since the bug is in production I don't have means to reproduce

**Comment by balazsorban44** [comment]
Hi, this sounds incorrect. Could you put together a minimal public reproduction if you cannot share your source code? :pray:
